docs/pages/versions/unversioned/sdk/status-bar.mdx

```diff
@@ -5,7 +5,7 @@ description: A library that provides the same interface as the React Native Stat
 sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-status-bar'
 packageName: 'expo-status-bar'
 iconUrl: '/static/images/packages/expo-status-bar.png'
-platforms: ['android', 'ios', 'web']
+platforms: ['android', 'ios', 'web', 'tvos']
```

Comment:
Status bar code will compile and run on tvOS, but no status bar will show. We might want to note that somewhere.

Reply:
I think it's worth adding a callout on this page.

Reply:
I'll follow up on this in a separate PR.